rhscl-postgresql-apb/README.md

```diff
@@ -0,0 +1,25 @@
+rhscl-postgresql-apb
+======================
+
+An apb for deploying [PostgreSQL](https://www.postgresql.org).
+
+## What it does
+* Deploys 1 postgresql container.
+
+## Requirements
+
+## Parameters
+* NAMESPACE: Optional, default 'postgresql-demo-apb', Namespace to deploy the cluster in.
+* POSTGRESQL_DATABASE, default 'admin', Postgresql database name.
+* POSTGRESQL_PASSWORD, default is a randomly generated string, Postgresql database password.
```

Comment:
No, the --env-file flag is added to docker run my-container --env-file vars.sh. The method that you just mentioned is another way to do this. You can build a simple RPC into the containers where they expect a file to be mounted to a particular location ... -v /etc/my,cnf:/usr/local/my.cnf. Then, a script runs that copies /usr/local/my.cnf to the location to be picked up by the process /etc/my.cnf. Or you could mount directly to where the process can pick it up ... -v /etc/my,cnf:/etc/my.cnf